src/JSONNestedNode.js

```diff
@@ -32,8 +32,10 @@ const styles = {
 export default class JSONNestedNode extends React.Component {
   defaultProps = {
     data: [],
+    nodeExpanded: false,
```

Comment:
that's a little confusing, since `nodeExpanded` should be a function. Also it's better to define it in `src/index`, not here, I think, like:
```
nodeExpanded: (keyName, data, level) => level === 0 // expands root by default
```